docs/pages/blog/2020-developer-survey-results.md

```diff
@@ -0,0 +1,314 @@
+  <tr><td>1</td><td>system - performance</td><tr>
+  <tr><td>1</td><td>system - rework breakpoints</td><tr>
+  <tr><th>3</th><th>components consistency</th><tr>
+  <tr><th>3</th><th>grid - improve</th><tr>
+</table>
+
+### Comparison with last year
+
+There are a couple of noticeable differences compared to last year.
+Some can be explained by our work, others by the evolution of the ecosystem.
+Each item is prefixed by the multiplication factor of the pain point for 2020 relative to 2019.
+
+Decreasing pain:
+- x0: Slider. No requests. The requirements are mostly met, especially with the introduction of the range feature.
+- x0.1: Strict mode. We fixed a lot of strict mode compatibility issues this year. However, since Create React App has made this mode a default, we have seen a lot more requests for it.
+- x0.1: Autocomplete. We added a new component, and have resolved a large number of issues opened since. This will be moved from lab to the core in v5.
+- x0.2: Fewer breaking changes. Only releasing minor versions under v4 for over a year helps a lot. However, we still need to be careful with CSS changes.
+- x0.3: Accessibility. We have been able to leverage GitHub issues opened by a11y experts, often coming from large companies using Material-UI at scale to improve it a lot this year.
+- x0.4: Material design. We didn't do much for it this year, at least not as much as we could have. Maybe the reduction is because fewer people care? It seems that we start to resonate more with developers building custom design systems.
+- x0.4: TypeScript. The continued migration of all the demos to TypeScript and of all the props to IntelliSense is paying off.
+- x0.5: Date picker. We did a lot for it this year. We probably still need the range feature, and to move it into the main repository (docs migration, etc.) for consistency.
+- x0.7: Performance. We didn't do much this year, so perhaps developers are leveraging React more effectively with virtualization, update pruning, etc?
+
+Growing pains:
+- ∞: Forms is a new item. It seems that we should at least work more closely with react-hook-form, formik, and react-final-form.
+- ∞: Charts is a new item. Material Design even has a page dedicated to [date visualization](https://material.io/design/communication/data-visualization.html).
+- x5: Custom themes.
+- x5: Simpler customization. We have improved customizability this year by introducing global class names and reducing the CSS specificity of some selectors. However, it seems that we are now tapping into a new audience. We need to do better.
+- x1.5: Animations.
+- x1.2: React native. We still have no plans for it. The [market is too small](https://npm-stat.com/charts.html?package=react-dom,react-native) to make it sustainable with our model.
+- x1.1: More components, the more we give, the more developers ask for. We will fully solve this with the enterprise version. The first early access will land this year.
```

Comment:
```suggestion
- x1.1: More components, the more we give, the more developer asks for. We will try to better solve this with the enterprise version. Both because it's the best model we have found that can sustain the developments of data-rich advanced components and because it allows us to invest back resources into the MIT side. The first early access will land this year.
```